Samples/AudioPreprocessing/WinUI/AudioPreprocessing/AudioPreprocessing/MainWindow.xaml

```diff
@@ -0,0 +1,29 @@
+﻿<Window
+    x:Class="AudioPreprocessing.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AudioPreprocessing"
+    xmlns:sys="using:System"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:vm="clr-namespace:AudioPreprocessing.ViewModel"
+    xmlns:model="using:AudioPreprocessing.Model"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d">
+
+    <StackPanel Margin="0,0,0,0" Background="White" Orientation="Vertical">
```

Comment:
I don't think you need to put all 0's for the margin. you can just leave it out right?

Reply:
Removed!